dace/runtime/include/dace/math.h

```diff
@@ -479,30 +479,29 @@ namespace dace
             return (T)std::pow(a, b);
         }
 
+        template<typename T, typename E>
+        DACE_CONSTEXPR DACE_HDFI T pow(const T& a, const E& b)
+        {
+            return (T)std::pow(a, (T)b);
```

Comment:
Is the cast really necessary? Since C++11, there is an overload for when the two arguments have different types (overload A): https://en.cppreference.com/w/cpp/numeric/math/pow

Shouldn't we just forward directly to that overload and let the standard library pick the implementation?

Reply:
Good idea.